csvs_to_sqlite/utils.py

```diff
@@ -39,7 +44,7 @@ def load_csv(filepath, separator, skip_errors, quoting, shape, encodings_to_try=
 def csvs_from_paths(paths):
     csvs = {}
 
-    def add_file(filepath):
+    def add_item(filepath, full_path=None):
```

Comment:
Having to have this second argument is a bit ugly. Right now I can't think of a nicer way of doing it though. I think the key of the dict is used as shortname for logging and as table name(?).

Most of the work of this function is counting up till it finds a unique key, which is why I decided against creatign a new `add_url()` which would duplicate most of `add_file()` but handle URLs.

What do you think?